src/views/ApartmentApp/index.js

```diff
@@ -52,209 +38,22 @@ export default class ApartApp extends Component {
       const profile = await user.getProfileInfo();
       this.setState({ userProfile: profile });
       this.setState({ isStu: String(profile.PersonType).includes('stu') });
-      if (this.state.isStu) {
-        let applicants = this.state.applicants;
-        applicants.push(profile);
-        this.setState({ applicants });
-      }
+      this.setState({ isFac: String(profile.PersonType).includes('fac') });
+      this.setState({ isAlu: String(profile.PersonType).includes('alu') });
```

Comment:
No, we do not need the the Alumni and Faculty ones specifically, it was just copied from several other pages on the site. I think I just didn't want to remove it yet until I was sure it would not be of use later. 
At most, we might need the student type, since we should not allow non-students to get the student application page